### PR TITLE
style: four eslint rules enabled

### PR DESCRIPTION
**Pull request by mightyiam**
- import/newline-after-import
- dot-notation
- import/first
- no-undef

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -16,10 +17,7 @@ module.exports = {
     '@typescript-eslint/restrict-plus-operands': 'off',
     '@typescript-eslint/strict-boolean-expressions': 'off',
     'comma-dangle': 'off',
-    'dot-notation': 'off',
-    'import/first': 'off',
```
Comment (paldepind) [inline .eslintrc.js]: Why was this removed❓

Comment (mightyiam) [reply .eslintrc.js]: @paldepind notice that most rules listed here are actually turning off rules that are turned on in the shareable config we extend from. So removing them is actually adding them.

**Comment by mightyiam** [comment]
Thank you for reviewing, @paldepind .
